### ISSUE TITLE
Disable markdown highlighting inside psalm tags

### ISSUE THREAD

**Issue by MaXal** [JetBrains/phpstorm-psalm-plugin#9]
Code to reproduce:
```
<?php

/**
 * @param array<int, array<string, int>> $arr
 */
function takesArray(array $arr) : void {
    if ($arr) {
        echo $arr[0][1]; // type should be string
    }
}
```

The parts `<int` and `<string` will be highlighted in green.

![image](https://user-images.githubusercontent.com/266747/86797689-4386cf80-c070-11ea-95be-44aac480bb01.png)

**Comment by MaXal** [comment]
The code looks very messy with the green background:
![image](https://user-images.githubusercontent.com/266747/94438792-e0549700-019f-11eb-9b1f-c1a4c72b812b.png)
![image](https://user-images.githubusercontent.com/266747/94438832-eba7c280-019f-11eb-8b63-b50212d70737.png)
![image](https://user-images.githubusercontent.com/266747/94438848-f1050d00-019f-11eb-9d25-e77bdf1d067c.png)

**Comment by wbars** [comment]
Fixed by commit in intellij 039d11be

**Comment by wbars** [comment]
Disabled for generic array
